io.github.pejuko.follamac.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "io.github.pejuko.follamac",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.node20"
+  ],
+  "command": "run.sh",
+  "build-options": {
+    "append-path": "/usr/lib/sdk/node20/bin",
+    "env": {
+      "NPM_CONFIG_LOGLEVEL": "info"
+    }
+  },
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "follamac",
+      "buildsystem": "simple",
+      "build-options": {
+        "env": {
+          "XDG_CACHE_HOME": "/run/build/follamac/flatpak-node/cache",
+          "YARN_CACHE_FOLDER": "/run/build/follamac/flatpak-node/npm-cache"
+        }
+      },
+      "build-commands": [
+        "HOME=$PWD yarn config --offline set yarn-offline-mirror $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror",
+        "yarn install --offline",
+        "yarn --offline build",
+        "yarn --offline dist --x64 --linux --dir",
```

Comment:
if you pass `x64` then for sure things won't work on anything else.

Reply:
should be fixed